ui/src/common/SongInfo.js

```diff
@@ -49,10 +49,17 @@ export const SongInfo = (props) => {
     updatedAt: <DateField source="updatedAt" showTime />,
     playCount: <TextField source="playCount" />,
     bpm: <NumberField source="bpm" />,
+    publisher: <TextField source="publisher" />,
```

Comment:
This makes the (already space-constrained) songlist even more cramped, wouldn't it be better from a UI clutter point of view to have this column only available in the (sparser) albums list and leave it out for songs?

Reply:
These are not fields in the SongList, it is the information that is shown when you select "Get Info" from the context menu. I think this is the proper place to display this info

Reply:
I agree with you that the songlist is already very packed but the UI makes it very easy to hide. And I depends on everyone personal usage. In my case, I don't care about updatedAt or Channels fields, so I just hide those.

But if you prefer that, I am not completely against

Reply:
Maybe I was not clear in my previous comment: these fields are NOT SongList columns, they are shown here:
<img width="647" alt="Screenshot 2023-05-18 at 1 21 50 PM" src="https://github.com/navidrome/navidrome/assets/331353/57cf3131-6765-4a16-8f44-65ab48634f5e">

So this change is completely fine. And in the SongList you added it off by default so it is also ok.